source/assets/stylesheets/styles/_blog_article.scss

```diff
@@ -0,0 +1,279 @@
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-top: 15px;
+  }
+
+  @include breakpoint($blog-container-width) {
+    margin: 0 auto;
+    max-width: $blog-container-width;
+    padding-bottom: 50px;
+  }
+}
+
+.blog-article__content {
+  box-sizing: border-box;
+
+  p,
+  ul,
+  ol {
+    font-weight: 100;
+    line-height: 1.5;
+    padding-bottom: 26px;
+  }
+
+  a {
+    color: $article-link-text-color;
+    font-weight: 400;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  strong {
+    font-weight: 400;
+  }
+
+  em {
+    font-style: italic;
+  }
+
+  blockquote {
+    &:before {
+      background-color: $blockquote-white-background-color;
+      content: '';
+      display: block;
+      height: 30px;
+    }
+
+    background-color: $blockquote-background-color;
+    margin: -30px;
+
+    p {
+      padding: 30px 60px;
+    }
+
+    margin-bottom: 26px;
+  }
+
+  img {
+    box-sizing: border-box;
+    padding: 20px 0;
+    width: 100%;
+  }
+
+  li {
+    font-weight: 100;
+    line-height: 1.4;
+    margin-left: 3px;
+
+      &:before {
+        content: "•";
+        padding-right: 5px;
+      }
+  }
+
+  ol li {
+    list-style-type: decimal;
+  }
+
+  h2 {
+    font-size: 1.3em;
+    font-weight: bold;
+    line-height: 1.2;
+    padding-bottom: 20px;
+  }
+
+  h3 {
+    color: $header-brand-color;
+    font-size: 1.2em;
+    font-weight: 100;
+    line-height: 1.2;
+    padding-bottom: 14px;
+  }
+}
+
+.blog-author {
+  background-color: $author-section-background-color;
+  border-top: 3px solid;
+  border-color: $author-section-border-color;
+}
+
+.blog-author__container {
+  background-color: $author-container-background-color;
+
+  @include breakpoint($blog-container-width) {
+    max-width: $blog-container-width;
+    margin: 0 auto;
+  }
+}
+
+.blog-author__image {
+  float: left;
+  margin-left: 10px;
+  margin-top: 20px;
+  width: 10%;
+
+  img {
+    display: inline;
+    height: 100px;
+    width: 100px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    margin-left: 50px;
+    margin-top: 50px;
+  }
+}
+
+.blog-author__right-section {
+  float: left;
+  margin-left: 70px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  width: 65%;
+
+  @include breakpoint($tablet-breakpoint) {
+    margin-top: 50px;
+  }
+}
+
+.blog-author__name {
+  color: $author-name-color;
+  font-size: 0.9em;
+  padding-bottom: 30px;
+}
+
+.blog-author__bio {
+  font-weight: 100;
+  line-height: 1.5;
+}
+
+.blog-author__clear {
+  clear: both;
+}
+
+.blog-comments__container {
+  border-top: 3px solid;
+  border-color: $comments-section-border-color;
+  background-color: $comments-container-background-color;
+  padding: 30px;
+
+  @include breakpoint($blog-container-width) {
+    margin: 0 auto;
+    max-width: $blog-container-width;
+    padding: 0;
+  }
+}
+
+.disqus-thread {
+  @include breakpoint($blog-container-width) {
+    padding: 40px 60px 40px;
+  }
+}
```

Comment:
We have 4 block styles in this file (`_blog_article`):

- `blog-header`
- `blog-article`
- `blog-author`
- `blog-comments`

so could probably move them out into 3 other files for consistent organisation (`.disqus-thread` could possibly be renamed to `.blog-comments__disqus-thread` and we just add the new class name to the HTML)